src/standup/standup.controller.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  ParseIntPipe,
+  Post,
+  Query,
+} from '@nestjs/common';
+import { ApiQuery, ApiTags } from '@nestjs/swagger';
+import { CreateStandupDTO } from './dto/CreateStandup.dto';
+import { StandupService } from './standup.service';
+
+@ApiTags('Standup')
+@Controller('standup')
+export class StandupController {
+  constructor(private readonly standupService: StandupService) {}
+
+  @Post()
+  createStandup(@Body() body: CreateStandupDTO) {
+    return this.standupService.create(body);
+  }
+
+  @Get()
+  findAllStandups() {
+    return this.standupService.findAll();
+  }
+  @Get('search')
```

Comment:
Search is now get

Reply:
Great thanks 👍 

Btw I think a blank line is needed between the methods